wiki/testing-guide.md

```
@@ -203,11 +206,13 @@ Note: The kernel tests have module dependency issues in this environment - they
 ### Test Module
 
 The test module is located at:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)